weaviate/collection/classes/config.py

```diff
@@ -32,18 +60,62 @@ class DataType(str, Enum):
     PHONE_NUMBER = "phoneNumber"
 
 
-class VectorIndexType(str, Enum):
+class _VectorIndexType(str, Enum):
     HNSW = "hnsw"
 
 
 class Tokenization(str, Enum):
+    """The available tokenization methods for text properties in Weaviate.
+
+    Attributes:
+        `WORD`
+            Tokenize by word.
+        `WHITESPACE`
+            Tokenize by whitespace.
+        `LOWERCASE`
+            Tokenize by lowercase.
+        `FIELD`
+            Tokenize by field.
+    """
+
     WORD = "word"
     WHITESPACE = "whitespace"
     LOWERCASE = "lowercase"
     FIELD = "field"
 
 
 class Vectorizer(str, Enum):
+    """The available vectorizers in Weaviate.
+
+    See the [docs](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules) for more details.
+
+    Attributes:
+        `NONE`
+            No vectorizer.
+        `TEXT2VEC_OPENAI`
+            OpenAI's embeddings module for text.
```

Comment:
Propose `For using text2vec-openai` and similar. Currently sounds a bit like OpenAI built the module.

The same for all below.

Reply:
Also, just to be a pedant the `text2vec_xxx` type components are `modules`, each of which may use one of a set of `models`. I don't know if's worth find/replacing though, just more of an FYI.